Add inert global attribute (#19090)
mdn/content: files/en-us/web/html/global_attributes/inert/index.md

---
title: inert

> slug: Web/HTML/Global_attributes/inputmode

Reviewer: ```suggestion
slug: Web/HTML/Global_attributes/inert
```

tags:
  - Attribute
  - Global attributes
  - HTML
  - Input
  - Reference
  - Text
  - Web
  - global
  - inert
browser-compat: html.global_attributes.inert
---

{{HTMLSidebar("Global_attributes")}}

The **`inert`** [global attribute](/en-US/docs/Web/HTML/Global_attributes) instructs the browser to disregard input events sent by the user, including focus-related events, as well as events generated by assistive technologies.

Specifically, it prevents the following actions to happens:

- It prevents the {{domxref("Element/click_event", "click")}} event from being thrown when the user clicks on the element.
- It prevents the {{domxref("Element/focus_event", "focus")}} event to be raised, by preventing the focus to be given to the element.

> - It prevents assistive technologies raised on this element.

Reviewer 2: Not sure what this means? It is hidden from the a11y tree

Reviewer: What about:
```suggestion
- It hides the element and its content from assistive technologies by excluding them from the accessibility tree.
```

Reviewer: I went ahead and committed this, but we can still work on line 25, if you want.

---
title: inert
slug: Web/HTML/Global_attributes/inputmode
tags:
  - Attribute
  - Global attributes
  - HTML
  - Input
  - Reference
  - Text
  - Web
  - global
  - inert
browser-compat: html.global_attributes.inert
---

> 

Reviewer 3: @teoli2003 note, after we start using Prettier, Prettier will enforce an empty line after the front matter. I suggest we start accepting empty lines here.

> 

Reviewer: They are fixed by the DailyMarkdownLint scripts.

> 

Reviewer 3: When you search for `---\n\n` you'll see a lot of files with empty lines here—I think it's usually better to add an empty line.

> 

Reviewer: I'm fine this, then.

{{HTMLSidebar("Global_attributes")}}

The **`inert`** [global attribute](/en-US/docs/Web/HTML/Global_attributes)  in HTML can allows for the browser to disregard input events by the user.

> Specifically, it allows for many cool uses such as,
> 
> •prevent the click event from generating
> 
> •prevent implicit focus
> 
> •prevent events from assistive technologies

Reviewer: ```suggestion
Specifically, it prevents the following actions to happens:

- It prevents the {{domxref("Element/click_event", "click")}} event from being thrown when the user clicks on the element.
- It prevents the {{domxref("Element/focus_event", "focus")}} event to be raised, by preventing the focus to be given to the element.
- It prevents assistive technologies raised on this element.
```

---
title: inert
slug: Web/HTML/Global_attributes/inert
tags:
  - Attribute
  - Global attributes
  - HTML
  - Input
  - Reference
  - Text
  - Web
  - global
  - inert
browser-compat: html.global_attributes.inert
---

{{HTMLSidebar("Global_attributes")}}

The **`inert`** [global attribute](/en-US/docs/Web/HTML/Global_attributes) instructs the browser to disregard input events sent by the user, including focus-related events, as well as events generated by assistive technologies.

> Specifically, it prevents the following actions to happens:
> 
> - It prevents the {{domxref("Element/click_event", "click")}} event from being thrown when the user clicks on the element.
> - It prevents the {{domxref("Element/focus_event", "focus")}} event to be raised, by preventing the focus to be given to the element.
> - It hides the element and its content from assistive technologies by excluding them from the accessibility tree.

Reviewer 3: ```suggestion
Specifically, it does the following:

- It prevents the {{domxref("Element/click_event", "click")}} event from being thrown when the user clicks on the element.
- It prevents the {{domxref("Element/focus_event", "focus")}} event from being raised, by preventing the element from gaining focus.
- It hides the element and its content from assistive technologies by excluding them from the accessibility tree.
```
